site/content/blogs/apprunner-waf.en.md

```diff
@@ -0,0 +1,82 @@
+# Using WAF With AppRunner in Copilot
+
+Posted On: Feb 08, 2023
+
+**Siddharth Vohra**, Software Development Engineer, AWS App Runner
+
+You can now associate your AWS WAF WebACLs with your App Runner service - all in Copilot!
+
+AWS Web Application Firewall (WAF) helps you monitor requests that are forwarded to your web applications and allows you to control access to your content. To do this, you need to create WAF Web Application Control List (Web ACL) which contains many rule options for your application. You can add multiple rules and create a Web ACL. Once you’ve created a Web ACL through AWS WAF Console or the AWS CLI, copy and save the Web ACL ARN. Now follow the steps below.
+
+ Step 1: If you don’t already have an App Runner service created/deployed, run `copilot init` to create and configure an App Runner service. (Do not deploy it just yet!)  
+
+Step 2: Go to the service's directory (users/\<your username\>/copilot/\<your App Runner service name\>). If you don't have a folder called 'addons' in this directory already, create a new folder and name it 'addons'. Now, go to the AR-WAF Addon discussion (Hyperlink coming soon) and follow the steps.  Your folders would now look like this:  
```

Comment:
Why do we need them to go to "AR-WAF Addon discussion"? Isn't this blog post specific enough?
```suggestion
Step 2: Go to the service's directory (users/\<your username\>/copilot/\<your App Runner service name\>). If you don't have a folder called 'addons' in this directory, create a new 'addons' folder. Now, go to the AR-WAF Addon discussion (Hyperlink coming soon) and follow the steps.  Your folders would now look like this:  
```

Reply:
This blog post is indeed quiet specific but the show-and-tell section is where other addons sit too and it might be a good area to increase visibility. Also, we could also then get direct feedback from users regarding the addon.